build.yaml

```diff
@@ -196,15 +238,26 @@ common_cli:
     # SSL on our caching box.
     # The --trusted-host option is only available with pip 7.
     #
-    parse_logs pip install --upgrade pip
+    if [ "\$(is_ubuntu)" == "true" ] || [ "\$(is_centos)"  == "true" ]; then
+        PIP_ADDITIONAL_OPTIONS=" -i \${PIP_INDEX_URL} --trusted-host \${TRUSTED_HOST}"
+        PARSE_LOGS="parse_logs "
+    fi
+    if [ "\$(is_osx)" == "true" ]; then
```

Comment:
these variables are not set on OSX (yet), however I would rather have a method to explicitly define what variables I set for different OS